language-design.md

```diff
@@ -106,6 +158,12 @@ you do so? If not, why not?
 
 **Response**
 
+These experiences are not necessarily at odds with each other.  Based off the Quorum [documentation]( http://quorumlanguage.com/syntax.php) and the above quote about Applescript, a key difference between the two languages seems to be that, while both languages incorporate natural language, Applescript is loose in its syntax while Quorum is strict.  This is where Applescript goes wrong.  Cook states that it was the “syntactic variations and flexibility” that confused users, not the fact that the words themselves were taken from natural language.  The potential for confusion when loose syntax is allowed is summarized well in the following stackexchange [thread](http://programmers.stackexchange.com/questions/49623/should-programming-languages-be-strict-or-loose):
```

Comment:
I think this is a huge point, and why your point below resonates with me. It's dangerous to expect users to treat a programming language like natural language, but then poorly execute/interpret nuances and subtleties of speech when the programmer tries to write them in.